en/messages.json

```diff
@@ -594,5 +594,79 @@
     "popupHolidayPromo2023CTA": {
         "message": "Get 1 year of Relay Premium",
         "description": "DO NOT TRANSLATE \"Relay Premium\"."
+    },
+    "popupPhoneMasksMetaForwardingDescription": {
+        "message": "Relay is currently forwarding all phone calls and text messages to your true phone number.",
+        "description": "DO NOT TRANSLATE \"Relay\"."
+    },
+    "popupPhoneMasksMetaBlockingDescription": {
+        "message": "Relay is blocking all phone calls and text messages—you will not receive anything from your phone number mask.",
```

Comment:
```suggestion
        "message": "Relay is blocking all phone calls and text messages — you will not receive anything from your phone number mask.",
```

https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-hyphens-and-dashes-c9